examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
+    match body {
+        Body::Sized(b, _)  => {
+            let mut data: Vec<u8> = vec![];
+            b.read(&mut data).expect("Read from response failed");
+            return Some(data);
+        },
+        Body::Chunked(b, _) => {
+            let mut data: Vec<u8> = vec![];
+            b.read(&mut data).expect("Read from response failed");
+            return Some(data);
+        }
+    }
+}
+
+fn test_query_file<T> (path: &str, status: Status, content: T)
+    where T: Into<Option<Vec<u8>>>
```

Comment:
Can't we just make this `Into<Option<&'static str>>` and then pass the filename and do the file reading here?